### PR TITLE
Add script to clean up snapshots history

### PR DESCRIPTION
**Pull request by Ndpnt**

### CODE REVIEW

File: scripts/rewrite/rewrite.js [hunk 1]
```diff
@@ -0,0 +1,207 @@
+// ecrire une doc
```
Comment (clementbiron) [inline scripts/rewrite/rewrite.js:1]: Is the comment useful?

Comment (Ndpnt) [reply scripts/rewrite/rewrite.js:1]: Nope :D

File: scripts/rewrite/rewrite.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+const fs = fsApi.promises;
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+export const SNAPSHOTS_SOURCE_PATH = path.resolve(__dirname, '../../', config.get('rewrite.snapshotsSourcePath'));
+export const SNAPSHOTS_TARGET_PATH = path.resolve(__dirname, '../../', config.get('history.snapshotsPath'));
+
+const README_PATH = path.resolve(__dirname, './initFiles/snapshot-readme.md');
+const LICENSE_PATH = path.resolve(__dirname, './initFiles/snapshot-license');
+
+const EMPTY_TOS_BACK_CONTENT = `<!DOCTYPE html><html><head></head><body>
+
```
Comment (clementbiron) [inline scripts/rewrite/rewrite.js:20]: Need an empty line?

Comment (Ndpnt) [reply scripts/rewrite/rewrite.js:20]: Yep, to be the exact same content of ToSBack empty documents

**Pull request merged**